demo/src/App.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+/* package doesn't exist on npm yet */
+/* eslint import/no-extraneous-dependencies: 0 */
```

Comment:
This is not that important if we plan to publish this to npm. But I'd prefer to have the demo working, simply adding the local dependency with `yarn add file:..`.
In this case I would also mention in the demo readme that you need to build the main package before running the demo.

Reply:
Yeah. Good idea! Thanks! I was using `yarn link` because I use this demo for development, but if you want just to try `file:...` is a great option.